### ISSUE TITLE
Cannot redeclare Mockery_3_Mockery_1_PHPStan_Type_ObjectType_PHPStan_Type_ObjectType_Mockery_1_PHPStan_Type_ObjectType_PHPStan_Type_ObjectType::mockery_init()

### ISSUE THREAD

**Issue by mfn** [nunomaduro/larastan#141]
- Larastan Version: 5.7.3
- `--level` used: default

### Description
- Installed phpstan 0.13.0
- Installed larastan 0.2.10
- ran `php artisan code:analyse`

Got this:
```
$ php artisan code:analyse
  56/525 [▓▓░░░░░░░░░░░░░░░░░░░░░░░░░░]  10%PHP Fatal error:  Cannot redeclare Mockery_3_Mockery_1_PHPStan_Type_ObjectType_PHPStan_Type_ObjectType_Mockery_1_PHPStan_Type_ObjectType_PHPStan_Type_ObjectType::mockery_init() in /vagrant/api.project.io/vendor/mockery/mockery/library/Mockery/Loader/EvalLoader.php(34) : eval()'d code on line 905

   Symfony\Component\Debug\Exception\FatalErrorException  : Cannot redeclare Mockery_3_Mockery_1_PHPStan_Type_ObjectType_PHPStan_Type_ObjectType_Mockery_1_PHPStan_Type_ObjectType_PHPStan_Type_ObjectType::mockery_init()

  at /vagrant/api.project.io/vendor/mockery/mockery/library/Mockery/Loader/EvalLoader.php: 34
  30:         if (class_exists($definition->getClassName(), false)) {
  31:             return;
  32:         }
  33:
  34:         eval("?>" . $definition->getCode());
  35:     }
  36: }
  37:


   Whoops\Exception\ErrorException  : Cannot redeclare Mockery_3_Mockery_1_PHPStan_Type_ObjectType_PHPStan_Type_ObjectType_Mockery_1_PHPStan_Type_ObjectType_PHPStan_Type_ObjectType::mockery_init()

  at /vagrant/api.project.io/vendor/mockery/mockery/library/Mockery/Loader/EvalLoader.php: 34
  30:         if (class_exists($definition->getClassName(), false)) {
  31:             return;
  32:         }
  33:
  34:         eval("?>" . $definition->getCode());
  35:     }
  36: }
  37:
```

### Laravel code where the issue was found:

I don't know where this comes from, the larastan docs say by default it's run for `app` only where I don't use mockery.

Note: I've an existing phpstan only setup which does not exhibit this crash (tuned for level 6):
```json
    "phpstan": "vendor/bin/phpstan analyse --level=6 --configuration=phpstan.neon app"
```
and `phpstan.neon` contains
```
parameters:
  bootstrap: vendor/autoload.php
  ignoreErrors:
    # Lots of errors I'm ignoring
```
But it does not work with 0.10.3 yet (in the sense I haven't fixed all errors, hence I'm trying laratan and see if I can reduce this).

I can't even say if this is really a larastan issue or just a accumulation of stuff.

Any pointers highly appreciated!

**Comment by nunomaduro** [comment]
@mfn This issue got addressed on the version `v0.2.11`. Can you test it?

**Comment by mfn** [comment]
@nunomaduro awesome, works!
